tracpro/templates/polls/poll_update.html

```diff
@@ -0,0 +1,49 @@
+{% extends "smartmin/update.html" %}
+
+{% load smartmin %}
+
+{% block post-fields %}
+  {{ questions_formset.management_form }}
```

Comment:
I wonder if it would be helpful to include the rapidpro_name, not editable, but just informational, in case it's different from the local poll name?